framework/quickstart.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Quick Start"
+description: "Novu Framework, the Code-first notification workflow engine. Build advanced workflows while giving your non-technical teammates complete control over content and behavior."
+---
+
+import { EchoTerminal } from '/snippets/echo-terminal.mdx';
+
+<Note>
+    **Novu Framework is currently in public Alpha.** Need help? Have a suggestion? Join our [Discord community](https://discord.gg/novu).
+</Note>
+
+## Introduction to Novu Framework
+
+Novu Framework allows you to write notification workflows in your codebase. Workflows are functions that execute business logic and use your preferred libraries for email, SMS, and chat generation. You can use Novu Framework with [React.Email](https://react.email/), [MJML](https://mjml.io/), or any other template generator.
+
+<EchoTerminal />
+
+## Try it now
+
+To get started, open your terminal and launch the Dev Studio.
```

Comment:
This is not relevant anymore. We will replace novu-labs CLI with the novu cli. Till then, it is better if we remove this section.

Reply:
There are a lot of references to the dev studio that will need to be affected, lets target it when we have an end-2-end flow in place and are happy with it.